activelearning.py

```diff
@@ -9,6 +9,8 @@
 from env import Env
 from gflownet import GFlowNet
 from querier import Querier
+from utils.logger import Logger
+import wandb
```

Comment:
We don't use wandb in this file do we ?

Reply:
Right I am sorry. I added it for test purposes and forgot to remove the import line. Will do it now